src/main/java/com/puppycrawl/tools/checkstyle/DefaultConfiguration.java

```diff
@@ -103,8 +102,9 @@ public String getProperty(String propertyName) throws CheckstyleException {
 
     @Override
     public Configuration[] getChildren() {
-        return children.toArray(
-                EMPTY_CONFIGURATION_ARRAY);
+        return Arrays.stream(children.toArray(new Configuration[0]))
```

Comment:
Why are we converting a list to an array and streaming it, instead of directly streaming the array?

Reply:
If not converting the list to array, yields new pitest error.